tests/e2e/nebulacluster_test.go

```diff
@@ -45,6 +46,7 @@ type (
 		InstallWaitNCOptions []envfuncsext.NebulaClusterOption
 		LoadLDBC             bool
 		UpgradeCases         []ncTestUpgradeCase
+		BackupCases          []ncBackupCase
```

Comment:
Why not use `UpgradeFunc ` in `UpgradeCases` ?

Reply:
I think it's better to add a new category of BackupCases rather then the upgrade function since the backup tests don't upgrade the cluster in any way. They'll only take a backup of the cluster and some may do a restore.